seatunnel-e2e/seatunnel-flink-connector-v2-e2e/connector-jdbc-flink-e2e/src/test/resources/container-license-acceptance.txt

```diff
@@ -0,0 +1 @@
+mcr.microsoft.com/mssql/server:2022-latest
```

Comment:
Why do you need to add this file？

Reply:
[https://www.testcontainers.org/modules/databases/mssqlserver/](mssqlserver)
[https://hub.docker.com/_/microsoft-mssql-server#environment-variables](microsoft-mssql-server)
the image `mcr.microsoft.com/mssql/server` need to accept an EULA .